### PR TITLE
Describe how to run CST using Docker Compose

### PR DESCRIPTION
**Pull request by nettoclaudio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
 
 This project was designed to allow security scans out of the box. You would use it since at project's pipeline through where your imagination would go.
 
+## Running CST
+
+This section outlines the required steps to run CST anywhere. The easiest way to
+deploy the CST it's using Docker Compose. Thus, you should install
```
Comment (ggarnier) [inline README.md]: `The easiest way to deploy CST is using Docker Compose`

File: README.md [hunk 2]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
...
+## Running CST
+
+This section outlines the required steps to run CST anywhere. The easiest way to
+deploy the CST it's using Docker Compose. Thus, you should install 
+[Docker][Docker Install] and [Docker Compose][Docker Compose Install] before to
+follow the instructions.
+
+### Configuring CoreOS Clair
+
+All configurations about CoreOS Clair is on `etc/clair.cfg.yml` file. That's a
```
Comment (ggarnier) [inline README.md]: `All configurations about CoreOS Clare are on...`

File: README.md [hunk 3]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
...
+self-explaned config file came from [Clair's repository][Clair Repository],
```
Comment (ggarnier) [inline README.md]: `... config file from ...`

File: README.md [hunk 4]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
...
+see more details there.
+
+**WARNING**:
+Unfortunately, Clair doesn't handle the database connection string via environment
+variables yet. So, we hardcoded the database credentials on its config file.
+We appreciate you should change those credentials on: `etc/clair.cfg.yml`
```
Comment (ggarnier) [inline README.md]: `You should change...`

File: README.md [hunk 5]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
...
+(line 23); and `docker-compose.yml` (envs `POSTGRES_USER`, `POSTGRES_DB` and
+`POSTGRES_PASSWORD`).
+
+### Certificate
+
+For start the CST's web server, you will need a certificate and its private key.
```
Comment (ggarnier) [inline README.md]: `To start CST web server, ...`

File: README.md [hunk 6]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
...
+Those files must be named `cert.pem` and `key.pem`, respectively, residing in
```
Comment (ggarnier) [inline README.md]: `... respectively, in the ``.certs`` dir.`

File: docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    container_name: cst-server
+    command:
+    - server
+    - --database
+    - mongodb://${CST_DATABASE_USER}:${CST_DATABASE_PASSWORD}@database.cst.local:27017/cst?authSource=admin
+    - --cert-file
+    - /etc/ssl/cst/cert.pem
+    - --key-file
+    - /etc/ssl/cst/key.pem
+    depends_on:
```
Comment (ggarnier) [inline docker-compose.yml:55]: Shouldn't this container also depend on `cst-worker`?

**Review by ggarnier — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 7 above)
Comment (nettoclaudio) [reply docker-compose.yml:55]: `cst-server` doesn't talk with `cst-worker` directly. All messages between them are intermediated by the queue service (mongodb).

**Review by ggarnier — APPROVED** [review]

**Review by andrestc — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -7,4 +7,59 @@ CST, stands for Container Security Testing, is a project to provide container se
...
+
+To start the CST web server, you will need a certificate and its private key.
+Those files must be named `cert.pem` and `key.pem`, respectively,  in the
+`.certs` dir.
+
+In a local env, you can generate a self-signed certificate running the command
+below, for instance.
+
+```bash
+$ make generate-self-signed-certificate
```
Comment (andrestc) [inline README.md:40]: Maybe https://github.com/FiloSottile/mkcert is worth checking out for this?

**Pull request merged**